python/cudf/cudf/_lib/csv.pyx

```diff
@@ -87,7 +90,7 @@ cpdef read_csv(
     if delimiter is None:
         delimiter = sep
 
-    nvtx_range_push("CUDF_READ_CSV", "purple")
+    nvtx_range_push("CUDF_READ_CSV", "PURPLE")
```

Comment:
Lowercase strings would be preferable as they will match matplotlib's color specification (https://matplotlib.org/3.1.0/gallery/color/named_colors.html).

Can we hadle the conversion of uppercase (or mixed case) strings to lowercase somewhere? (possibly within `nvtx_range_push`)?

Reply:
Oh. Can do.